test/ERC6551/ERC6551RegistryTest.js

```diff
@@ -0,0 +1,200 @@
+        const account = await Account.deploy();
+        await registry.deployed();
+        await account.deployed();
+        return {registry, account};
+      }
+
+      describe("Deployment", function () {
+        it("Should return valid name and symbol", async function () {
+          const {registry, account} = await loadFixture(
+              deployContracts);
+          expect(await registry.name()).to.equal("Private Aave AM");
+          expect(await registry.symbol()).to.equal("PACM");
+        });
+      });
+
+      describe("Create Account,Mint NFT", function () {
+        it("Should create account and mint nft", async function () {
+          const {registry, account} = await loadFixture(
+              deployContracts);
+
+          // params to create a new account
+          const chainId = BigInt(
+              (await ethers.provider.send('eth_chainId', [])));
+          const tokenAddress = ethers.constants.AddressZero;
+          const initData = [];
+          const salt =0;
+          const [acc1, acc2, acc3] = await ethers.getSigners();
+
+          //creating an account with acc1
+          const create = await registry.createAccount(account.address, chainId,
+              tokenAddress,
+              0, initData);
+
+          // await expect(create).to.emit(registry, "AccountCreated").withArgs("",
+          //     account.address, chainId, tokenAddress,
+          //     maxUint, salt);
+
+          await expect(create).to.emit(registry, "AccountCreated");
+
+          // getting the created account
+          const accountAddress = await registry.account(account.address,
+              chainId, registry.address, 0, salt);
+
+          // validating the owner of the created account
+          const AC = await ethers.getContractFactory("DefaultERC6551Account");
+          const ac = await AC.attach(accountAddress);
+          expect(await ac.owner()).to.equal(acc1.address);
+
+          //validation the nft owner
+          expect(await registry.ownerOf(0)).to.equal(acc1.address);
+
+          //creating another account with acc1
+          const create2 = await registry.createAccount(account.address, chainId,
+              tokenAddress,
+              0, initData);
+
+          await expect(create2).to.emit(registry, "AccountCreated");
+
+          // getting the created account
+          const accountAddress2 = await registry.account(account.address,
+              chainId, registry.address, 1, salt);
+
+          // validating the owner of the created account
+          const AC2 = await ethers.getContractFactory("DefaultERC6551Account");
+          const ac2 = await AC2.attach(accountAddress2);
+          expect(await ac2.owner()).to.equal(acc1.address);
+
+          //validation the nft owner
+          expect(await registry.ownerOf(1)).to.equal(acc1.address);
+
+        });
+      });
+      describe("Create Account With Different NFT address", function () {
```

Comment:
Add an empty line before this. And make it consistent across all files. Use linter and prettier for formatting.